doc/sphinx/language/core/basic.rst

```diff
@@ -523,24 +523,24 @@ they appear after a boldface label.  They are listed in the
 Locality attributes supported by :cmd:`Set` and :cmd:`Unset`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The :cmd:`Set` and :cmd:`Unset` commands support the :attr:`local`,
-:attr:`global` and :attr:`export` locality attributes:
-
-* no attribute: the original setting is *not* restored at the end of
-  the current module or section.
-* :attr:`local` (or alternatively, the ``Local`` prefix): the setting
+The :cmd:`Set` and :cmd:`Unset` commands support the mutually
+exclusive :attr:`local`, :attr:`export` and :attr:`global` locality
+attributes:
+
+* if no attribute is specified: the original value of the setting is
+  restored at the end of the current module but it is *not* restored
+  at the end of the current section.
+* :attr:`local` (or the ``Local`` prefix): the setting
```

Comment:
```suggestion
* :attr:`local` (or the ``Local`` :n:`@legacy_attr`): the setting
```

and similar changes elsewhere

Reply:
I don't think it improves readability. Furthermore, there is the special case of `Export` which isn't a proper legacy attribute (as we've previously discussed) and which would break symmetry.